src/frontend/get-jira-configuration.ts

```diff
@@ -1,13 +1,15 @@
 import format from "date-fns/format";
 import moment from "moment";
-import { Subscription } from "../models";
-import SubscriptionClass from "../models/subscription";
+import SubscriptionClass, { SyncStatus } from "../models/subscription";
+import { RepoSyncState, Subscription } from "../models";
 import { NextFunction, Request, Response } from "express";
 import statsd from "../config/statsd";
 import { metricError } from "../config/metric-names";
 import { FailedInstallations } from "../config/interfaces";
+import { booleanFlag, BooleanFlags } from "../config/feature-flags";
+import Logger from "bunyan";
 
-function mapSyncStatus(syncStatus: string): string {
+function mapSyncStatus(syncStatus: SyncStatus = SyncStatus.PENDING): string {
```

Comment:
In which case this method will have to use the default parameter?

Reply:
`Subscription.syncStatus` is optional, we needed a way to handle in the off chance that it wasn't set yet, but almost certainly would be by this point as this is for current subscriptions which any new subscriptions is automatically synced and set to `PENDING`.  https://github.com/atlassian/github-for-jira/blob/main/src/models/subscription.ts#L166